rules/aircraft.yaml

```diff
@@ -1,3 +1,96 @@
+shad:
+	Inherits: ^Aircraft
+	Tooltip:
+		Name: Night Hawk
+		Description: Infantry Transport Helicopter\nUndetectable by radar.\n  Strong vs Infantry\n  Weak vs Vehicles, Aircraft
+	Health:
+		HP: 175
+	Armor:
+		Type: Light
+	Armament:
+		Weapon: BlackHawkCannon
+	Voiced:
+		VoiceSet: NighthawkVoice
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 20
+		Prerequisites: ~gaweap
+	Valued:
+		Cost: 1000
+	RevealsShroud:
+		Range: 7c0
+		Type: CenterPosition
+	-AppearsOnRadar:
+	AttackFrontal:
+		Voice: Attack
+	Aircraft:
+		CruiseAltitude: 3072
+		LandWhenIdle: true
+		LandableTerrainTypes: Clear, Rough, Road, DirtRoad
+		InitialFacing: 0
+		ROT: 10
+		Speed: 100
+		AltitudeVelocity: 120
+		Voice: Move
+		AirborneUpgrades: airborne
+		CruisingUpgrades: cruising
+		CanHover: True
+	Targetable@airborne:
+		TargetTypes: Air
+		UpgradeTypes: airborne
+		UpgradeMinEnabledLevel: 1
+	Targetable@ground:
+		TargetTypes: Ground
+		UpgradeTypes: airborne
+		UpgradeMaxEnabledLevel: 0
+	RenderSprites:
+	RenderVoxels:
+	WithVoxelWalkerBody:
+		TickRate: 1
+	Hovers@CRUISING:
+		UpgradeTypes: cruising
+		UpgradeMinEnabledLevel: 1
+		OffsetModifier: -45
+	Cargo:
+		Types: Infantry
+		MaxWeight: 5
+		PipCount: 5
+		UnloadVoice: Move
+	SpawnActorOnDeath:
+		Actor: shadhusk
+
+shadhusk:
+	Tooltip:
+		Name: Night Hawk Down
+		GenericName: Destroyed Helicopter
+	Armor:
+		Type: Light
+	Health:
+		HP: 140
+	Aircraft:
+		ROT: 5
+		Speed: 140
+		AirborneUpgrades: airborne
+		CanHover: True
+		Voice: Move
+	FallsToEarth:
+		Spins: True
+		Moves: False
+		Explosion: UnitExplode
```

Comment:
Those three values seem to be the default too: https://github.com/OpenRA/OpenRA/blob/d416138cec825752fe3e971d1990504e7c730707/OpenRA.Mods.Common/Traits/Air/FallsToEarth.cs#L21-L24